From 84cc323948da5f41daec290c97c7fa6a18979837 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Fri, 24 Sep 2010 14:46:12 +0800
Subject: [PATCH 179/435] ENGR00131878 enable IMX_NEEDS_DEPRECATED_SYMBOLS

Enable IMX_NEEDS_DEPRECATED_SYMBOLS compiler flag for
module_test build. It can fix the compiler error caused
by SDMA_BASE_ADDR in MX3 platform.

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 module_test/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/module_test/Makefile b/module_test/Makefile
index f16d87e..f79dae9 100644
--- a/module_test/Makefile
+++ b/module_test/Makefile
@@ -41,7 +41,7 @@ sahara_test_driver-objs := km_test.o $(SAHARA_TEST_SOURCE)run_tests.o \
 
 obj-m += $(module_list-y)
 
-EXTRA_CFLAGS := -DLINUX_KERNEL
+EXTRA_CFLAGS := -DLINUX_KERNEL -DIMX_NEEDS_DEPRECATED_SYMBOLS
 
 ## NOTE: The following should be removed. They are directly requesting kernel
 ##       source.
-- 
1.8.0

